jetty-http/src/main/java/org/eclipse/jetty/http/pathmap/ServletPathSpec.java

```diff
@@ -168,52 +168,47 @@ else if (servletPathSpec.startsWith("*."))
     public String getPathInfo(String path)
     {
         // Path Info only valid for PREFIX_GLOB types
-        if (group == PathSpecGroup.PREFIX_GLOB)
+        switch (group)
         {
-            if (path.length() == (specLength - 2))
-            {
+            case ROOT:
```

Comment:
This comment should really go on the PathSpecGroup class, but as it isn't part of the PR I'm putting this comment here instead.

The comments at PathSpecGroup class line 38-42 need fixing I think as they don't contain enough explanation and have contradictory values.